### DISCUSSION TITLE
Migrating Third-party Bridges

### DISCUSSION THREAD

**Discussion by jonathan-neufeld-asurion** [apache/pekko#476] [Q&A] [answered]
Has there been any effort (or indication thereof) to migrate certain third-party bridges from Akka to Pekko? the following artifacts in particular...

- akka-stream-alpakka
- akka-stream-kafka
- tapir-swagger-ui-akka-http
- tapir-akka-http-server
- tapir-redoc-akka-http
- akka-http-backend (com.softwaremill.sttp.client)
- akka-discovery-kubernetes-api
- akka-http-circe (de.heikoseeberger)

**Comment by luksow** [comment]
https://github.com/hseeberger/akka-http-json/issues/780

**Comment by pjfanning** [comment, marked as answer]
* akka-stream-kafka equivalent is https://github.com/apache/incubator-pekko-connectors-kafka
* akka-stream-alpakka equivalent is https://github.com/apache/incubator-pekko-connectors
* akka-discovery equivalent is https://github.com/apache/incubator-pekko-management
* tapir issue and linked PRs - https://github.com/softwaremill/tapir/issues/2884
* sttp PR - https://github.com/softwaremill/sttp/pull/1781
